### DISCUSSION TITLE
بروزرسانی و رفع مشکل اسکنر اندروید و ویندوز

### DISCUSSION THREAD

**Discussion by ghost** [MortezaBashsiz/CFScanner#599] [Q&A]
درود به مرتضای عزیز
اول از همه به خاطر تمام زحمتی که تو این مدت کشیدی و می‌کشی صمیمانه ازت سپاسگزارم ❤️

دوم اینکه اخیرا اسکنر اندروید از کار افتاده و اسکنر ویندوز هم فقط without fronting جوابگو هست.
اگر برات ممکنه، ازت خواهش میکنم که این مشکل رو برطرف کنی ...
مخصوصا اندورید، از این جهت که قابل دسترس تر از بقیه هست برای ما ...
بازم ازت ممنونم و قدردان زحماتت هستم ...

**Comment by kingofkings2023** [comment]
سلام، بنده هم به نوبه خودم از مرتضی عزیز خواهش می کنم در صورت امکان اپدیت جدید رو منتشر کنن و مث همیشه توی اسکنرها در صدر یکه تازی کنن
به شخصه به تجربه هیچ اسکنری رو دست cfscanner ندیدم
